src/Sulu/Component/Rest/ListBuilder/Metadata/Resources/schema/list-2.0.xsd

```diff
@@ -57,6 +57,11 @@
         <xs:attributeGroup ref="propertyAttributeGroup"/>
     </xs:complexType>
 
+    <xs:complexType name="countDistinctPropertyType">
```

Comment:
Maybe not needed if we change the exist adapter. Lets wait for the response of the other core teams members.